CRM/Utils/JSON.php

```diff
@@ -94,4 +94,38 @@ public static function encodeDataTableSelector($params, $sEcho, $iTotal, $iFilte
     return $sOutput;
   }
 
+  /**
+   * This function is used to encode data for new dataTable plugin v1.10 and greater
+   * @return string
+   *
+   */
+  public static function encodeDataTable($params, $iTotal, $iFilteredTotal, $selectorElements) {
```

Comment:
@deepak-srivastava this function does not look necessary.
It seems to be duplicating the legacy code in encodeDataTableSelector which is deprecated. In 4.7+ we simply use `CRM_Utils_JSON::output()` to output json.